jobs/rep_windows/spec

```diff
@@ -45,6 +45,9 @@ properties:
     description: "Array of name:absolute_path pairs representing root filesystems preloaded onto the underlying garden"
     default:
       - windows2012R2:/tmp/windows2012R2
+  diego.rep.sidecar_rootfs_path:
+    description: "absolute_path representing the root filesystem used for sidecar processes (ie. "/var/vcap/packages/cflinuxfs4/rootfs.tar"). Must be one of the preloaded_rootfses paths. Leaving the default empty string is ok, as it will then select the first of he preloaded_rootfses"
+    default: ""
```

Comment:
Is this something you added, or something we removed from the main branch that is about to get re-added?